changelogs/fragments/ospfv2_module_docfix.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
extra please remove

Reply:
removed

Reply:
add back the changelog to the PR remove extra ---